manifest.yaml

```diff
@@ -24,7 +24,7 @@ box:
     strategy:
       download: bucket
       project: catalyst-api
-      commit: aba02096b6ed17521f72c1943fa58a126a9f76fe
+      commit: b5a530d514d23c06b5ad64abfc5a5f29dd637fd0
```

Comment:
It's my rollback of `CDN redirect` feature - https://github.com/livepeer/catalyst-api/pull/827